core/ir/x86/decode.c

```diff
@@ -797,6 +797,10 @@ read_evex(byte *pc, decode_info_t *di, byte instr_byte,
     info = *ret_info;
 
     CLIENT_ASSERT(info->type == EVEX_PREFIX_EXT, "internal evex decoding error");
+    if (info->type != EVEX_PREFIX_EXT) {
```

Comment:
Instead of all these `if..LOG` which won't actually log anything b/c they won't be reached in debug and in release LOG is a nop, use `IF_DEBUG(info = *ret_info)` (or `DEBUG_DECLARE(const instr_info_t *info = *ret_Info)`.

Reply:
Done.